pkg/arvo/mar/kiln/jump.hoon

```diff
@@ -0,0 +1,43 @@
+::
+::::  /hoon/jump/kiln/mar
+  ::
+/-  h=hood
+|_  jum=jump:h
+::
+++  grow
+  |%
+  ++  noun  jum
+  ++  json
+    =,  enjs:format
+    |^  ^-  ^json
+    ?-    -.jum
```

Comment:
minor nit:
the `%+  frond  'foo'` can be moved above the wutlus for brevity, i.e.
```hoon
%+  frond  -.jum
```